gpus/example1/example1.cu

```diff
@@ -28,9 +25,9 @@ int main() {
 
 
     // we will create size element vectors with entries of zero
-    vector<float> a(size, 0);
-    vector<float> b(size, 0);
-    vector<float> c(size, 0);
+    thrust::host_vector<float> a(size, 0);
```

Comment:
Since "host" and "device" in the slides are changed to "CPU" and "GPU", I think it should be mentioned somewhere what _host_\_vector is.

Reply:
It's my plan to port the slides I showed the other day to README docs.

Reply:
They should contain this information.